packages/flutter_tools/test/commands.shard/permeable/test_test.dart

```diff
@@ -62,7 +62,8 @@ void main() {
 
     testUsingContext('report which user created widget caused the error - no flag', () async {
       Cache.flutterRoot = '../..';
-      return _testFile('print_user_created_ancestor_no_flag', automatedTestsDirectory, flutterTestDirectory);
+      return _testFile('print_user_created_ancestor_no_flag', automatedTestsDirectory, flutterTestDirectory,
+         extraArguments: const <String>['--no-track-widget-creation']);
```

Comment:
we should run tests with --track-widget-creation by default. It provides a better experience when your tests have rendering errors as you can figure out where in your source code you likely triggered the error.

Reply:
Tests use the same track-widget-creation setting so they are on by default too. This test case is just testing the stack traces for twc being off, which is why I needed to switch the flag

Reply:
sgtm